src/pages/categories/categories-add/categories-add.js

```diff
@@ -517,14 +518,24 @@ const CategoriesAdd = ({ id, editMode }) => {
           </Grid>
         </Grid>
       </FormControl>
-      <Button
-        variant='contained'
-        color='primary'
-        className={classes.saveBtn}
-        onClick={editMode ? handleCategoryEdit : handleCategorySave}
-      >
-        {config.buttonTitles.titleGenerator(editMode, isMain)}
-      </Button>
+
+      <div className={classes.controlsBlock}>
+        <StandardButton
+          id='back'
+          title={config.buttonTitles.GO_BACK_TITLE}
+          variant='outlined'
+          onClickHandler={() => dispatch(push(config.routes.pathToCategories))}
```

Comment:
Make sense to move this dispatch to a separated handler function

Reply:
done